includes/class-patches.php

```diff
@@ -110,6 +111,13 @@ public static function add_patterns_menu_link() {
 		add_submenu_page( 'edit.php', 'manage_patterns', __( 'Patterns' ), 'edit_posts', 'edit.php?post_type=wp_block', '', 2 );
 	}
 
+	/**
+	 * Add a menu link in WP Admin to easily edit and manage pattern categories.
+	 */
+	public static function add_pattern_categories_menu_link() {
+		add_submenu_page( 'edit.php', 'manage_pattern_categories', __( 'Pattern Categories' ), 'edit_posts', 'edit-tags.php?taxonomy=wp_pattern_category', '', 3 );
```

Comment:
:)

```suggestion
		add_submenu_page( 'edit.php', 'manage_pattern_categories', __( 'Pattern Categories', 'newspack-plugin' ), 'edit_posts', 'edit-tags.php?taxonomy=wp_pattern_category', '', 3 );
```

Reply:
🤦‍♀️ 😆 !! Thanks Adam -- fixed in [c9cbc3e](https://github.com/Automattic/newspack-plugin/pull/2932/commits/c9cbc3e07f1d32de8f7bed4a5fbbc4ed97f1ea7f)!